### DISCUSSION TITLE
cli needs module 'tailwindcss/plugin'

### DISCUSSION THREAD

**Discussion by jbdemonte** [tailwindlabs/tailwindcss#4302] [Help]
Hi

**[edit]** Sorry, I thought being on [tailwindcss-aspect-ratio](https://github.com/tailwindlabs/tailwindcss-aspect-ratio/issues/new/choose)

Is there a way to use the plugin `tailwindcss-aspect-ratio` with the command line version:

    npx tailwindcss-cli@latest build -o src/tailwind.css -c ./tailwind.config.js

I get the error: ` Cannot find module 'tailwindcss/plugin'`

**Comment by mbrodt** [comment]
I'm getting the same issue (unable to use plugins purely using the CLI)

I'm guessing because the plugins module is only exported as a part of the main tailwindcss package, but curious to know if there's a workaround (that doesn't involve installing Tailwind/Postcss etc. in the traditional way)
